packages/utils/upgrade/src/modules/project/project.ts

```diff
@@ -13,15 +13,15 @@ import type { Version } from '../version';
 import type { Codemod } from '../codemod';
 import type { Report } from '../report';
 import type {
-  Project as ProjectInterface,
-  AppProject as AppProjectInterface,
-  PluginProject as PluginProjectInterface,
+  IAppProject,
+  IPluginProject,
   FileExtension,
   MinimalPackageJSON,
   RunCodemodsOptions,
+  IProjectBase,
 } from './types';
 
-export class Project implements ProjectInterface {
+export class Project implements IProjectBase {
```

Comment:
I need to find where we had this discussion (I think it's internal).
But let's keep our distance from the C# interface notation to stay consistent with the rest of the codebase (well, except DTS, but we'll need to worry about that later)

Reply:
Yeah, the problem is it interferes with the class imports of the same name. Do we have guidelines on that? I think we shouldn't be using classes at all, but I didn't want to refactor the entire thing for this little feature.